packages/maps/src/components/result/addons/components/helper.js

```diff
@@ -0,0 +1,64 @@
+/* global google */
+function reduce(obj, fn, acc) {
+	return Object.keys(obj).reduce((newAcc, key) => fn(newAcc, obj[key], key), acc);
+}
+
+function forEach(obj, fn) {
```

Comment:
Are these polyfills?

Reply:
@bietkul No, these are not polyfills. Just some generalized wrapper around the function.